src/components/Layout/Layout.tsx

```diff
@@ -212,10 +256,14 @@ export const Layout = forwardRef(function Layout(
               </View>
 
               <View className="layout-main">
-                <Flex as="main" className="main">
-                  <Breadcrumbs route={pathname} platform={currentPlatform}/>
+                <Flex as="main" className={`main${hasTOC ? ' main--toc' : ''}`}>
+                  <Breadcrumbs route={pathname} platform={currentPlatform} />
                   {children}
                 </Flex>
+                {hasTOC && tocHeadings.length > 0 ? (
+                  <TableOfContents headers={tocHeadings} />
+                ) : null}
```

Comment:
On mobile, should the TOC appear above the `main` content? This can be a fast-follow though, not sure what the other think.

I think for this to work we would need to wrap the `main` and TOC in it's own flex container right?

Reply:
Yeah I was a little torn on this. It makes sense to me that you'd probably want immediate access to the ToC so you aren't forced to keyboard navigate the entire page content just to get to the ToC links; which renders them not all that useful to keyboard users.

We could put a "Table of contents >" skip-to-nav kind of link at the top of the page and clicking it would navigate you down to the ToC? That way you could bypass the main content and go directly to the ToC? What do you think about that?

Reply:
That sounds okay, this isn't blocking though so no worries about getting it in now. What was the original reason to have it display under?